test/e2e/storage/drivers/csi.go

```diff
@@ -257,10 +258,9 @@ func InitMockCSIDriver(driverOpts CSIMockDriverOpts) testsuites.TestDriver {
 				"", // Default fsType
 			),
 			Capabilities: map[testsuites.Capability]bool{
-				testsuites.CapPersistence:  false,
-				testsuites.CapFsGroup:      false,
-				testsuites.CapExec:         false,
-				testsuites.CapVolumeLimits: true,
```

Comment:
The mock drivers does support volume limits, but the test isn't running for this drivers anyway (it doesn't support persistence).

Reply:
I'm confused why we would drop this capability, then

Reply:
To clarify: I dropped because I found it misleading. It took me a while to understand why a test with this capability wouldn't run in any job.